lib/Reader.php

```diff
@@ -22,6 +22,12 @@ class Reader extends XMLReader {
     use ContextStackTrait;
 
     /**
+     * Reading indicator
+     * @var bool
+     */
+    public $reading = false;
```

Comment:
Changes it to private in https://github.com/fruux/sabre-xml/pull/110/commits/c8ca514ade1de7553c4e40797aaddf588a012652.